frontend/src/components/desktop_apps/AppShortcut.tsx

```diff
@@ -7,43 +7,58 @@ interface AppShortcutProps {
   setOpen: (at: AppType) => void;
   icon?: string;
   useSmaller?: boolean;
+  visible?: boolean;
 }
 
-export default function AppShortcut({ appType, setOpen, icon, useSmaller = false }: AppShortcutProps) {
+export default function AppShortcut({ appType, setOpen, icon, useSmaller = false, visible = true }: AppShortcutProps) {
   const appTextColor: string = useSelector((state: IGameState) => state.styleData.textColor.app);
   const appTextFont: string = useSelector((state: IGameState) => state.styleData.textFont.app);
-
   const shownIcon = `url(assets/app_icons/${icon ?? `${appType.toLowerCase()}`}.png)`;
   const size = useSmaller ? "64px" : "120px";
-  const appTextSize = useSmaller ? "18px" : "20px";
+  const appTextSize = setSize(useSmaller);
```

Comment:
What's the difference between `size` and `appTextSize`?

Reply:
Size refers to the size of the app's itself, white appTextSize refers to the size of the text underneath the app (ex.store, settings, ect.).